CODE_OF_CONDUCT.rst

```diff
@@ -90,7 +90,7 @@ Violating these terms may lead to a permanent ban.
 Attribution
 ***********
 
-This Code of Conduct is adapted from the `Contributor Covenant <https://www.contributor-covenant.org>`_, version 2.0, available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+This Code of conduct is adapted from the `Contributor Covenant <https://www.contributor-covenant.org>`_, version 2.0, available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
```

Comment:
Why keep "Code" capitalized, but not "Conduct"?

Reply:
Good point. I did think about it and thought one might consider it a proper noun, thus capitalizing the first word.